tests/fixtures/api/devices/specs/_.GET.json

```diff
@@ -0,0 +1,14 @@
+{
+  "code": 200,
+  "data": [
+    {
+      "id": "00000000-0000-0000-0000-000000000001",
+      "created_at": "2020-01-01T00:00:00+00:00",
+      "updated_at": "2020-01-01T00:00:00+00:00",
+      "device_type": "QPU",
```

Comment:
This should be FRESNEL I think

Reply:
Fresnel is not yet a device in our DeviceType enum. QPU however, is.

Reply:
Fixed since we don't use the device type enum.